prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/Annotation.yml

```diff
@@ -0,0 +1,125 @@
+text:
+  type: STRING_ALL
+  valueOf: CWE
+  generateList: true
+  expressionType: HL7Spec
```

Comment:
Using `type: STRING_ALL` will concat all elements in CWE adding a comma delimiter